shub/image/test.py

```diff
@@ -1,13 +1,10 @@
-import sys
-
 import click
 
 from shub import exceptions as shub_exceptions
 from shub.config import load_shub_config, list_targets_callback
 from shub.image import utils
 
-if sys.version_info >= (3, 0):
-    long = int
+long = int
```

Comment:
We can remove this line altogether and replace `isinstance(size, (int, long))` with `isinstance(size, int)` later on.